server/database/schema.sql

```diff
@@ -3089,9 +3089,8 @@ e'[
    "LO2: Exposure to and analysis of poorly accessible design: Examine a software application that doesn\’t properly accommodate accessibility in regards to identity (Analysis)",
```

Comment:
Can someone remind me the purpose of this file? Seems weird that we're storing the databases' data in GitHub. Or is this something else.

@dm9965 @heathermoses

Reply:
Good question, the schema file is used to spin up a local copy of the database for local development through docker. Correct me if I'm wrong @dm9965